Misc/NEWS.d/next/Build/2022-06-20-15-15-11.gh-issue-90656.kFBbKe.rst

```diff
@@ -0,0 +1 @@
+Add platform triplets for LoongArch64
```

Comment:
```suggestion
Add platform triplets for ``LoongArch64``:

- ``loongarch64-linux-gnusf``
- ``loongarch64-linux-gnuf32``
- ``loongarch64-linux-gnu``

Patch by Zhang Na.
```

Also add a similar entry to Doc/whatsnew/3.12.rst.

Reply:
The What's New category should be "Build Changes", IMO.

Reply:
Thanks, I will modify based on your suggestions.

Reply:
@erlend-aasland After modifying *.rst, error occurred in test DOC CI test, `make V=1 -C Doc/ SPHINXOPTS="-q" SPHINXERRORHANDLING="-W --keep-going" html` need "do it twice", I have verified on PC.  What's the reason for CI error? What shoud I do?

Reply:
It seems Sphinx is picky about lists. I fixed the PR to use `*` instead of `-`. Let's see what the CI says now. I also copied the NEWS entry verbatim to What's New, and fixed the formatting (IMO, using monospaced font for the triplets is not needed; also, I used the architecture name from the Wiki page.)